go/hash/hash.go

```diff
@@ -107,25 +121,32 @@ func Parse(s string) Hash {
 	return r
 }
 
+// Less compares two hashes returning whether this Hash is less than other.
 func (r Hash) Less(other Hash) bool {
 	return bytes.Compare(r.digest[:], other.digest[:]) < 0
 }
 
+// Greater compares two hashes returning whether this Hash is greater than other.
 func (r Hash) Greater(other Hash) bool {
+	// TODO: Remove this
```

Comment:
Yes. I didn't want to change the API in this PR.

Having Less and Greater is pretty silly. I'll open an issue.